cicd/api-test/api-test.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# configs for dev server
+TOKEN=$(KNOX_MACHINE_AUTH=$(hostname) knox get teletraan:dev:rodimusapitoken)
```

Comment:
I'm updating the token to connect to teletraanservice dev. Because we can't write to integ, we can only read and integ is a production service, we won't attach it to that.